### PR TITLE
Pass through AWS config

### PR DESCRIPTION
**Pull request by diversemix**

### CODE REVIEW

**Review by Cooryd — APPROVED** [review]

**Review by petereast — APPROVED** [review]
no

**Review by giorgiosironi — APPROVED** [review]

File: storage/docker-compose.yml [hunk 1]
```diff
@@ -16,6 +16,9 @@ services:
     command: ["bash", "-c", "yarn && yarn start"]
     environment:
       PG_HOST: database
+      AWS_ACCESS_KEY_ID: ${AWS_ACCESS_KEY_ID}
+      AWS_SECRET_ACCESS_KEY: ${AWS_SECRET_ACCESS_KEY}
+      AWS_REGION: ${AWS_REGION}
```
Comment (giorgiosironi) [inline storage/docker-compose.yml:21]: bucket name?

Comment (diversemix) [reply storage/docker-compose.yml:21]: @petereast ... doesn't the bucket name come from the namespace?

Comment (petereast) [reply storage/docker-compose.yml:21]: /cc @Cooryd Eventually it'll be derived from the config but at the moment it's hard coded. It's probably worth knowing that the namespace we're using is `someNameSpace` and the bucket we're using is `storage-service-demo`

Comment (diversemix) [reply storage/docker-compose.yml:21]: @giorgiosironi - you happy with this or I can make these pull from env's too as part of this PR ? (or another?)

Comment (giorgiosironi) [reply storage/docker-compose.yml:21]: as long as we know the name of the bucket, it can be created

Comment (petereast) [reply storage/docker-compose.yml:21]: Yeah, I'm happy, the bucket name is hard-coded to `storage-service-demo` - I'm going to merge this now

**Pull request merged**